apps/learneth/src/pages/Home/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faChevronRight,
+  faChevronDown,
+  faPlayCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import Markdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import remarkGfm from 'remark-gfm';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks';
+import RepoImporter from '../../components/RepoImporter';
+import './index.css';
+
+function HomePage(): JSX.Element {
+  const [openKeys, setOpenKeys] = React.useState<string[]>([]);
+
+  const isOpen = (key: string) => openKeys.includes(key);
+  const handleClick = (key: string) => {
+    setOpenKeys(
+      isOpen(key)
+        ? openKeys.filter((item) => item !== key)
+        : [...openKeys, key],
+    );
+  };
+
+  const dispatch = useAppDispatch();
+  const { list, detail, selectedId } = useAppSelector(
+    (state) => state.workshop,
+  );
+
+  const selectedRepo = detail[selectedId];
+
+  const levelMap: any = {
+    1: 'Beginner',
+    2: 'Intermediate',
+    3: 'Advanced',
+  };
+
+  useEffect(() => {
+    dispatch({
+      type: 'workshop/init',
+    });
+  }, []);
+
+  return (
+    <div className="App">
+      <RepoImporter list={list} selectedRepo={selectedRepo || {}} />
+      {selectedRepo && (
+        <div className="container-fluid">
+          {Object.keys(selectedRepo.group).map((level) => (
+            <div key={level}>
+              <div className="mb-2 border-bottom small">{levelMap[level]}:</div>
+              {selectedRepo.group[level].map((item: any) => (
+                <div key={item.id}>
+                  <div>
+                    <a
+                      href="#"
+                      className="arrow-icon"
+                      onClick={() => {
+                        handleClick(item.id);
+                      }}
+                    >
+                      <FontAwesomeIcon
+                        size="xs"
+                        icon={isOpen(item.id) ? faChevronDown : faChevronRight}
+                      />
+                    </a>
+                    <a
+                      href="#"
+                      className="workshop-link"
+                      onClick={() => {
+                        handleClick(item.id);
+                      }}
+                    >
+                      {selectedRepo.entities[item.id].name}
+                    </a>
+                    <Link
+                      to={`/list?id=${item.id}`}
+                      className="text-decoration-none float-right"
+                    >
+                      <FontAwesomeIcon icon={faPlayCircle} size="lg" />
```

Comment:
please use the same coding style as we do in other places.
example https://github.com/ethereum/remix-project/blob/master/libs/remix-ui/run-tab/src/lib/components/contractGUI.tsx#L366

Reply:
I resaved all files. Now it's using remix's coding style.

Reply:
I'm sorry I still see FontAwesomeIcon  in the code.

Reply:
> I'm sorry I still see FontAwesomeIcon in the code.

just removed. please check again. thanks @LianaHus